### ISSUE TITLE
Unblocking a blocked user does not work

### ISSUE THREAD

**Issue by SebastienDaniel** [modxcms/revolution#9664]
Askarii created Redmine issue ID 9664

Unblocking a user through manager works partially. However, the end result is that the user is still blocked.

I had to go fix the DB data to truely unblock the user.

When using "unblock" in user manager, it properly sets the DB value of "blocked" column to 0.
However, it does not change the DB value of "blockeduntil" value, which is what i had to manually set to 0 in order to unblock my user.

**Comment by pbowyer** [comment]
Not an issue, but a UI difference to what people may expect.

When unblocking a user there are two fields to adjust:
![image](https://cloud.githubusercontent.com/assets/89852/23562313/de8cd0a6-0039-11e7-935f-2fa3f7894493.png)

'Blocked' needs to be unticked AND the 'Blocked until' value deleted. Both of these can be actioned, but you may expect unticking the 'Blocked' checkbox to do everything.

Please close #modxbughunt

**Comment by OptimusCrime** [comment]
Perhaps, but it should be reported as its own issue.

@modxbot close
